### PR TITLE
Speech dev

### PR DESCRIPTION
**Pull request by DomHoney**
Merge Lawrences changes into master

### CODE REVIEW

File: src/ai_project/speech/text/test.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package ai_project.speech;
+package ai_project.speech.text;
```
Comment (Mattihew) [inline src/ai_project/speech/text/test.java]: Test classes should be in the test folder.

File: src/ai_project/speech/speechtotext/AudioInputFormat.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+package ai_project.speech.speechtotext;
+
+/// <summary>
+/// Voice output formats.
+/// </summary>
+public class AudioInputFormat {
```
Comment (Mattihew) [inline src/ai_project/speech/speechtotext/AudioInputFormat.java]: this should be an enum

File: src/ai_project/speech/texttospeech/AudioOutputFormat.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+package ai_project.speech.texttospeech;
+
+/// <summary>
+/// Voice output formats.
+/// </summary>
+public class AudioOutputFormat {
```
Comment (Mattihew) [inline src/ai_project/speech/texttospeech/AudioOutputFormat.java]: This should be an enum

File: src/ai_project/speech/texttospeech/Gender.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+package ai_project.speech.texttospeech;
+
+/// <summary>
+/// Gender of the voice.
+/// </summary>
+public class Gender
```
Comment (Mattihew) [inline src/ai_project/speech/texttospeech/Gender.java]: should be Enum

File: src/ai_project/speech/texttospeech/AudioOutputFormat.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        /// </summary>
+        public static final String Raw8Khz8BitMonoMULaw = "raw-8khz-8bit-mono-mulaw";
+        /// <summary>
+        /// raw-16khz-16bit-mono-pcm request output audio format type.
+        /// </summary>
+        public static final String Raw16Khz16BitMonoPcm = "raw-16khz-16bit-mono-pcm";
+        /// <summary>
+        /// riff-8khz-8bit-mono-mulaw request output audio format type.
+        /// </summary>
+        public static final String Riff8Khz8BitMonoMULaw = "riff-16khz-16bit-mono-pcm";
```
Comment (Mattihew) [inline src/ai_project/speech/texttospeech/AudioOutputFormat.java]: should this be equal to "riff-8khz-8bit-mono-mulaw" instead?

Comment (DomHoney) [inline src/ai_project/speech/texttospeech/AudioOutputFormat.java]: Looks like it, will fix

**Pull request merged**